xbmc/windowing/gbm/WinSystemGbm.cpp

```diff
@@ -330,9 +330,23 @@ bool CWinSystemGbm::SetHDR(const VideoPicture* videoPicture)
   if (!drm)
     return false;
 
+  auto connector = drm->GetConnector();
+  if (!connector)
+    return false;
+
   if (!videoPicture)
   {
-    auto connector = drm->GetConnector();
+    if (connector->SupportsProperty("Colorspace"))
+    {
+      auto [result, value] = connector->GetPropertyValue("Colorspace", "Default");
+      if (result)
+      {
+        CLog::Log(LOGDEBUG, "CWinSystemGbm::{} - setting connector colorspace to Default",
+                  __FUNCTION__);
+        drm->AddProperty(connector, "Colorspace", value);
```

Comment:
You do `drm->SetActive(true);` when setting the Colorspace property (presumably to trigger a modeset). Is one needed here when resetting the property?

Reply:
That's a good catch, added.